libwacom/libwacom-database.c

```diff
@@ -279,8 +323,88 @@ parse_stylus_id(const char *str, WacomStylusId *id)
 	return ret;
 }
 
+static char *
+string_or_fallback (GKeyFile *keyfile, const char *group,
+		    const char *key, const char *fallback) {
```

Comment:
```suggestion
		    const char *key, const char *fallback) 
		    {
```
including the others below too please